test/samples_test.cpp

```diff
@@ -306,6 +306,17 @@ TEST(Samples, SelfIntersect) {
   manifold::PolygonParams().processOverlaps = false;
 }
 
+TEST(Samples, GenericTwinBooleanTest7081) {
+  manifold::PolygonParams().processOverlaps = true;
```

Comment:
Is this necessary for this test to pass the CI?

Reply:
No, will remove.